components/transaction-operation/transaction-operation.module.css

```diff
@@ -0,0 +1,131 @@
+.background {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  background: rgba(0, 0, 0, 0.5);
+  z-index: 100;
+  display: block;
+  pointer-events: all;
+}
+
+.modalWrapper {
+  position: fixed;
+  z-index: 100;
+  background: white;
+  border-radius: 10px;
+  padding: 20px 10px;
+  width: 40%;
+  height: auto;
+  top: 30vh;
+  left: 30%;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.26);
+  display: block;
+  animation: fade-slide-in 0.3s ease-out forwards;
+}
+
+@keyframes fade-slide-in {
+  from {
+    transform: translateY(-30rem);
+  }
+  to {
+    transform: translateY(0);
+  }
+}
+
+.modalContent {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.modalActions {
+  padding: 1rem;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.label {
+  font-weight: bold;
+  margin: 0.5rem 0;
+  color: #464646;
+}
+
+.input[type='text'],
+.input[type='number'],
```

Comment:
NIT: Instead of assigning styles using attributes, why not give it a common class and use it accordingly.

Reply:
Sure I will do that changes, thanks @sumitd94